FIDO2_get_next_credential_rsp.md

```diff
@@ -0,0 +1,20 @@
+## FIDO2 Get Next Credential Response
```

Comment:
this file isn't referenced by anything and seems to be obsolete from what I see.

Reply:
Ditto